src/main/java/de/muenchen/isi/domain/service/AbfrageStatusService.java

```diff
@@ -417,4 +435,22 @@ private Map<AuthoritiesEnum, StatusAbfrageEvents> getAuthoritiesAndEventsMap() {
 
         return authoritiesAndEventsMap;
     }
+
+    private void throwStringLengthExceededExceptionWhenAnmerkungExceedsLength(UUID id, String anmerkung)
+        throws StringLengthExceededException, EntityNotFoundException {
+        InfrastrukturabfrageModel infrastrukturabfrage = this.abfrageService.getInfrastrukturabfrageById(id);
+        if (!anmerkung.isEmpty()) {
+            if (infrastrukturabfrage.getAbfrage().getAnmerkung() == null) {
+                infrastrukturabfrage.getAbfrage().setAnmerkung(anmerkung);
+            } else {
+                infrastrukturabfrage
+                    .getAbfrage()
+                    .setAnmerkung(infrastrukturabfrage.getAbfrage().getAnmerkung().concat("\n").concat(anmerkung));
+            }
+
+            if (infrastrukturabfrage.getAbfrage().getAnmerkung().length() > 255) {
+                throw new StringLengthExceededException("Die Anmerkung überschreitet das Zeichenlimit.");
```

Comment:
Bitte ändern in: "Es sind maximal 255 Zeichen erlaubt" (analog den anderen Meldungen in den DTO's)